docs/photoniq/vwrs/get-started-vwrs/get-started-vwrs-native.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 20
+title: Get Started with VWRs and Native Apps
+---
+
+This guide details how to integrate Macrometa PhotonIQ Virtual Waiting Rooms (VWRs) with your native applications. Unlike web-based non-native apps, native apps are developed for specific platforms using platform-specific programming languages. These include mobile apps on Android and iOS or desktop applications on Windows and macOS. Native apps require direct API interactions to manage user experiences effectively during high traffic periods.
+
+## Before You Begin
+
+Ensure you have completed the following prerequisites before starting the integration process:
+
+- **Coordinate with Macrometa Personnel:** Secure the necessary credentials and access rights by engaging with Macrometa personnel.
+- **Plan Your Waiting Room:** Identify the appropriate [queue type](../queue-types.md) for your application, determine which endpoints will incorporate the waiting room, and decide on the priority handling mechanisms.
+- **Collect Information:** Gather essential information, including API endpoints and expected traffic metrics, to facilitate a smooth setup.
+
+## Integration Overview
+
+Integrate VWRs with your native applications by following these steps:
+
+1. **API Key Creation:** Start by creating an API key to access the VWRs system, as this key will be essential for all API interactions. For instructions on creating an API key, refer to [Create an API Key](https://www.macrometa.com/docs/apiVwrs#/operations/createAPIKey).
+
+2. **Create and Configure a Domain:** Create a domain that hosts your waiting room. This step allows you to fine-tune how the waiting room operates and integrates with your site. For comprehensive guidance on domain configuration, refer to [Configure a Domain](../configure-domain.md).
+
+3. **API Usage for Queue Management:** Use the VWRs API to manage the user's position in the waiting room. This involves making periodic API calls to check the queue status and manage the user experience accordingly.
```

Comment:
You don't manage the position, the VWR tells you where you are in the position.  I could not come up with a clear alternative right now.